profiles/fscloud/variables.tf

```diff
@@ -53,49 +58,26 @@ variable "members" {
 
 variable "resource_tags" {
   type        = list(string)
-  description = "Optional list of tags to be added to created resources"
+  description = "Optional list of tags to be applied to the PostgreSQL database instance."
   default     = []
 }
 
-variable "allowlist" {
-  type = list(object({
-    address     = optional(string)
-    description = optional(string)
-  }))
-  default     = []
-  description = "Set of IP address and description to allowlist in database"
-}
-
-variable "configuration" {
```

Comment:
This still need to be a variable so consumers can configre. The question is whether we need to hard code any of these settings to be FSCloud compliant.